src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal.test.ts

```diff
@@ -0,0 +1,218 @@
+import '@testing-library/jest-dom';
+
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import _ from 'lodash/fp';
+import { h } from 'react-hyperscript-helpers';
+import { Ajax } from 'src/libs/ajax';
+import { getUser } from 'src/libs/state';
+import { DeepPartial } from 'src/libs/type-utils/deep-partial';
+import { GoogleWorkspace } from 'src/libs/workspace-utils';
+import ShareWorkspaceModal from 'src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal';
+import { AccessEntry, RawWorkspaceAcl } from 'src/pages/workspaces/workspace/WorkspaceAcl';
+import { asMockedFn } from 'src/testing/test-utils';
+
+jest.mock('src/libs/state', () => ({
+  ...jest.requireActual('src/libs/state'),
+  getUser: jest.fn(),
+}));
+
+type ModalMockExports = typeof import('src/components/Modal.mock');
+
+jest.mock('src/components/Modal', () => {
+  const mockModal = jest.requireActual<ModalMockExports>('src/components/Modal.mock');
+  return mockModal.mockModalModule();
+});
+
+jest.mock('src/libs/ajax');
+
+type AjaxExports = typeof import('src/libs/ajax');
+type AjaxContract = ReturnType<AjaxExports['Ajax']>;
+
+/**
```

Comment:
I think this comment is in the wrong place.

Reply:
it was